apiserver/params/params.go

```diff
@@ -57,6 +57,22 @@ type EntitiesResult struct {
 	Error    *Error   `json:"error,omitempty"`
 }
 
+// RemoveSpaceResults contains multiple RemoveSpace results.
+type RemoveSpaceResults struct {
+	Results []RemoveSpaceResult `json:"results"`
+}
+
+// RemoveSpaceResult contains entries if removing a space is not possible.
+// Constraints are a slice of entities which has constraints on the space.
+// Bindings are a slice of entities which has bindings on that space.
+// Error is filled if an error has occured which is unexpected.
+type RemoveSpaceResult struct {
+	Constraints        []Entity `json:"entities, omitempty"`
```

Comment:
idea from cmdline is:
- check whether each is set and create a corresponding error message.

```
errors.New("not possible because constraints: %q ...")
```